mkdocs/themes/bootstrap/css/base.css

```diff
@@ -3,7 +3,7 @@ body {
 }
 
 @media (min-width: 992px) {
-    h1[id]:before, h2[id]:before {
+    .headerfix, h1[id]:before, h2[id]:before, h3[id]:before, h4[id]:before, h5[id]:before, h6[id]:before {
```

Comment:
I think we should support all the h1-h6 tags in MkDocs. The .headerfix is more specific and should be added by users with an extra css file.

If we were to include it, we would need to start documenting the CSS classes users can insert into markdown and I don't want to do that.